### PR TITLE
V2 gui for the bootstrapper

### PR DESCRIPTION
**Pull request by roastario**

### CODE REVIEW

**Comment by gendal** [comment]
You spoilsport...  I was hoping there would be a few screenshots in here to whet the appetite!!

**Comment by tlil** [comment]
image

@gendal ^

**Comment by gendal** [comment]
nice!

**Comment by roastario** [comment]
Mike has gotten his pencil case out improved it quite a bit. 
screenshot from 2018-06-22 15-46-30
screenshot from 2018-06-22 15-47-37
screenshot from 2018-06-22 15-48-05

I think it's ready for primetime.

File: .idea/compiler.xml [hunk 1]
```diff
@@ -26,6 +26,8 @@
       <module name="canonicalizer_test" target="1.8" />
       <module name="client_main" target="1.8" />
       <module name="client_test" target="1.8" />
+      <module name="common_main" target="1.8" />
```
Comment (tlil) [inline .idea/compiler.xml]: Are all additions in this file needed? Or does it have any crud ?

**Pull request merged**